dream-web/controller/badges.ml

```diff
@@ -0,0 +1,73 @@
+type schema = {
+  schema_version : int; [@key "schemaVersion"]  (** always: 1 *)
```

Comment:
Couldn't the `[@default 1]` be added to enforce the condition?

Reply:
Should check if the `@default value` can be used to enforce a stricter typing here.